src/common/src/org/drftpd/util/CommonPluginUtils.java

```diff
@@ -1077,7 +1077,7 @@ public static String getPluginVersionForObject(Object obj) {
 				if (createInstance) {
 					T pluginInstance = null;
 					if (constructorSig == null) {
-						pluginInstance = pluginCls.newInstance();
+						pluginInstance = pluginCls.getDeclaredConstructor().newInstance();
 					} else {
 						pluginInstance = pluginCls.getConstructor(constructorSig).newInstance(constructorArgs);
 					}
```

Comment:
@epochyx could you check on this as well? Thanks

Reply:
pok @epochyx